schema/types/isolator.go

```diff
@@ -52,6 +77,24 @@ func (is *Isolators) GetByName(name ACIdentifier) *Isolator {
 	return nil
 }
 
+// ReplaceIsolatorsByName overrides matching isolator types with a new
+// isolator, deleting them all and appending the new one instead
+func (is *Isolators) ReplaceIsolatorsByName(newIs Isolator, oldNames []ACIdentifier) uint {
+	var numReplacements uint = 0
```

Comment:
That's its only way to signal if it actually replaced anything. While the *current* caller doesn't use it, I don't think it's a good idea to remove this observable return value completely.